### ISSUE TITLE
SASS import customization broken?

### ISSUE THREAD

**Issue by ollwenjones** [justindujardin/ng2-material#80]
Recently upgraded to beta.7 and so upbraded to 0.2.5 of ng2-material. 
Suddenly my custom color theme is gone, and the material icons no longer work.

Using webpack sass-loader to process these imports:
```
$md-font-url: "~./assets/material-icons/";
@import "~ng2-material/font/font.scss";
@import "~ng2-material/source/core.scss";
$md-primary: md-palette($md-blue-grey, 500, 100, 700, $md-contrast-palettes);
$md-accent: md-palette($md-blue, 500, 300, 700, $md-contrast-palettes);
$md-background: md-palette($md-grey, 100, 0, 200, $md-contrast-palettes);
@import "~ng2-material/source/components.scss";
```

Anybody else using webpack seen anything like this?

**Comment by justindujardin** [comment]
It works for me but I don't use webpack. Did you get this resolved?

**Comment by ollwenjones** [comment]
I got the icons working again by just adding the icon location variable
anymore. webpack reads the other import just fine I guess. I don't know
what the deal is with the theaming variables. I feel like I should get an
error or something, but it's as if they aren't getting picked up in the
subsequent imports.

The webpack sass-loader does use node-sass, which has a few particulars
that are different (such has having to add file extensions as you see in my
code-block), which maybe are contributing.

Wish I could say I've sorted it out, but honestly, color-scheme is a low
priority at this point in the project, haha. I'll be sure to put something
on this issue if I figure it out, though.
